Classes/common/CDTFetchChanges.h

```diff
@@ -0,0 +1,163 @@
+//
+//  CDTFetchChanges.h
+//  
+//
+//  Created by Michael Rhodes on 31/03/2015.
+//
+//
+
+#import <Foundation/Foundation.h>
+
+@class CDTDatastore;
+@class CDTDocumentRevision;
+
+/**
+ Read changes to a database.
+ 
+ CDTFetchChanges reads the changes to a database from a given sequence value. It provides
+ callbacks for updated and created documents (documentChangedBlock) and deleted documents
+ (documentWithIDWasDeletedBlock). It also provides a completion block for when all changes
+ have been received and processed.
+ 
```

Comment:
Could be more verbose (with examples?) about how using this class differs from the standard changes feed?

I think I remember you saying that this would suppress non-winning revisions? I see that you've documented this down below for the callback block, but perhaps it's worth a mention in the Class description.